cargo-dist-schema/src/lib.rs

```diff
@@ -610,12 +621,21 @@ impl DistManifest {
         if let Some(position) = self.releases.iter().position(|r| r.app_name == name) {
             &mut self.releases[position]
         } else {
-            let install_dir_env_var =
-                Some(name.to_ascii_uppercase().replace('-', "_") + "_INSTALL_DIR");
+            let env_app_name = name.to_ascii_uppercase().replace('-', "_");
+            let install_dir_env_var = format!("{env_app_name}_INSTALL_DIR");
+            let unmanaged_dir_env_var = format!("{env_app_name}_UNMANAGED_INSTALL");
+            let disable_update_env_var = format!("{env_app_name}_DISABLE_UPDATE");
+
```

Comment:
We should maybe also expose FLAT_INSTALL_LAYOUT as a raw option too? less certain of that

Reply:
How about we circle back around to publicizing this around the time we also give developers full control over the install layout?